modules/init.gradle

```diff
@@ -6,6 +6,14 @@ import org.gradle.internal.hash.HashValue
 if (System.properties["http.proxyHost"] == "squid.lax.liferay.com") {
 	logger.lifecycle "Using proxy {}:{}.", System.properties["http.proxyHost"], System.properties["http.proxyPort"]
 
+	File repositoryLocalDir = file("../.m2-tmp")
+
+	URI uri = repositoryLocalDir.toURI()
```

Comment:
Sure the [`uri`](https://docs.gradle.org/current/dsl/org.gradle.api.Project.html#org.gradle.api.Project:uri(java.lang.Object)) is not available?